src/core/types/Dao.ts

```diff
@@ -0,0 +1,38 @@
+import { Address, Optional } from './Common';
+import { GovernanceModule } from './Governance';
+import { Safe } from './Safe';
+import { SupportedChainId } from './Chains';
+
+export type GasTank = {
+  address: Optional<Address>;
+  enabled: boolean; // if enabled, address should hold a value
+};
+
+export type GovernanceCycle = {
+  firstStart: Date; // first ever cycle. Voting starts on cycle start
+  days: number; // Days
+};
+
+export type HatStream = {
+  hatId: string;
+  streamId: string;
+};
+
+export type Dao = {
+  chainId: SupportedChainId;
+  address: Address;
+  safe: Optional<Safe>;
+  name: Optional<string>;
+  proposalTemplatesCID: Optional<string>; // for ipfs proposal templates
+  governanceModules: Optional<GovernanceModule[]>; // null if Multi-sig
+  guardAddress: Address; // for subDAO, Freeze, parent DAO vote on FreezeVoting
+  fractalModuleAddress: Optional<Address>; // for subDAO only, parent DAO can force child DAO to make a transaction
```

Comment:
use `decent` instead of `fractal` since this is a brand new package? should we keep using old names?